queries.yaml

```diff
@@ -1740,6 +1741,58 @@ post_process:
       source_layer: buildings
       target_value_type: int
 
+  - fn: vectordatasource.transform.drop_properties
+    params:
+      source_layer: buildings
+      start_zoom: 0
+      end_zoom: 17
+      properties:
+        - landuse_kind
+      where: >-
+        landuse_kind not in ['aerodrome', 'airfield', 'cemetery', 'college', 'conservation', 'forest', 'garden', 'golf_course', 'hospital', 'national_park', 'nature_reserve', 'park', 'protected_area', 'runway', 'school', 'taxiway', 'university']
+
+  - fn: vectordatasource.transform.drop_properties
+    params:
+      source_layer: buildings
+      start_zoom: 0
+      end_zoom: 16
+      properties:
+        - kind_detail
+      where: >-
+        kind_detail not in ['airport', 'civic', 'church', 'college', 'commercial', 'fire_station', 'industrial', 'military', 'hospital', 'office', 'public', 'prison', 'retail', 'school', 'sports_centre', 'stadium', 'station', 'synagogue', 'train_station', 'university', 'warehouse']
+
+  - fn: vectordatasource.transform.drop_properties
+    params:
+      source_layer: buildings
+      start_zoom: 0
+      end_zoom: 16
+      properties:
+        - building_material
+        - layer
```

Comment:
Actually we need `layer` and `location` else **underground** transit buildings will show up very early, and underground building parts will extrude like underground mushrooms.

Another option is to exclude these features from tiles at early zooms, though that would negatively impact transit map styles.

Down tip of Manhattan island:

![image](https://user-images.githubusercontent.com/853051/153308565-09142d49-d464-4a26-a927-4649d467ecfc.png)

And Grand Central and Penn Station in midtown, also at zoom 15:

<img width="501" alt="image" src="https://user-images.githubusercontent.com/853051/153308671-e8e78e67-456e-4ef1-b01a-efda4ca86116.png">

Reply:
removed in the next commit